changelogs/CHANGELOG-1.0.md

```diff
@@ -127,10 +44,52 @@ We're excited to release our first alpha for v1.0! Please take it for a spin in
 #### Validation
 * Configs for Azure, AWS, and GCP are now checked for invalid or extra keys, and the server is halted if any are found
 
+### Download
+- https://github.com/heptio/velero/releases/tag/v1.0.0
+
+### Container Image
+`gcr.io/heptio-images/velero:v1.0.0`
+
+### Documentation
+TODO
```

Comment:
What documentation are you intending to put here?

Reply:
Just a link to the 1.0-tagged docs on velero.io